src/app/planner/containers/planner.component.ts

```diff
@@ -122,27 +123,37 @@ export class PlannerComponent implements OnInit {
     public drop(event) {
         const selectedSessionId = this.selectedSessionId;
         this.latestEvent = event;
-
-        if (!this.confirmationDialogOpen) {
-            this.confirmationDialogRef = this.openConfirmationDialog();
+        let hearingPartId = event.detail.jsEvent.target.getAttribute('data-hearingid');
+        let isMultiSession = this.hearingParts.find(hp => hp.id === hearingPartId).multiSession;
+
+        if (!isMultiSession) {
+            if (!this.confirmationDialogOpen) {
+                this.confirmationDialogRef = this.openConfirmationDialog();
+                this.confirmationDialogRef.afterClosed().subscribe(confirmed => {
+                    this.confirmationDialogOpen = false;
+                    if (confirmed) {
```

Comment:
3rd depth of 'if' indentation is a sign of potential problems, can this be reworked?
for example 
```
if (!this.confirmationDialogOpen) {
                this.confirmationDialogRef = this.openConfirmationDialog();
                this.confirmationDialogRef.afterClosed().subscribe(confirmed => {
                    this.confirmationDialogOpen = false
```
this fragment is in the end done twice.
I'm pretty sure (but not 100%) that you can attach to 'this.confirmationDialogRef.afterClosed()' multiple times, so prepare once and if it's not multi subscribe again?

Additionally, it's usually better to do positive checks like 'if (isNotMultisession)' though sometimes not possible

Reply:
Done